web-app/src/app/shared/components/form-item/form-item.component.html

```diff
@@ -0,0 +1,287 @@
+        <nz-tag [nzColor]="item.status == 0 ? 'processing' : 'error'">{{
+          item.status == 0 ? ('monitor.collector.status.online' | i18n) : ('monitor.collector.status.offline' | i18n)
+        }}</nz-tag>
+        <span nz-icon nzType="bug" nzTheme="outline" style="margin-right: 8px"></span>
+        <nz-tag nzColor="processing">{{ item.name }}</nz-tag>
+        <nz-tag nzColor="processing">{{ item.ip }}</nz-tag>
+        <nz-tag [nzColor]="item.mode == 'private' ? 'warning' : 'success'">
+          {{ item.mode == 'private' ? ('collector.mode.private' | i18n) : ('collector.mode.public' | i18n) }}
+        </nz-tag>
+      </nz-option>
+    </nz-select>
+    <ng-template #collectorTemplate let-selected>
+      <nz-tag nzColor="processing">{{ selected.nzLabel }}</nz-tag>
+    </ng-template>
+  </nz-form-control>
+</nz-form-item>
+
+<nz-form-item *ngIf="item.type === 'intervals'">
+  <nz-form-label nzSpan="7" [nzRequired]="item.required" [nzFor]="item.field" [nzTooltipTitle]="item.tooltip"
+    >{{ item.name }}
+  </nz-form-label>
+  <nz-form-control nzSpan="8">
+    <nz-input-number
+      [(ngModel)]="value"
+      (ngModelChange)="onChange($event)"
+      [nzMin]="extra.interval_type === 'push' ? 1 : 30"
+      [nzMax]="604800"
+      [nzStep]="extra.interval_type === 'push' ? 1 : 60"
+      name="intervals"
+      id="intervals"
+    >
+    </nz-input-number>
+    <nz-tag style="margin-left: 6px">{{ 'common.time.unit.second' | i18n }}</nz-tag>
+  </nz-form-control>
+</nz-form-item>
+
+<nz-form-item *ngIf="item.type === 'tags-selection'">
+  <nz-form-label nzSpan="7" [nzRequired]="item.required" [nzFor]="item.field" [nzTooltipTitle]="item.tooltip"
+    >{{ item.name }}
+  </nz-form-label>
+  <nz-form-control nzSpan="8">
+    <nz-tag
+      *ngFor="let tag of value; let i = index"
+      [nzMode]="tag.type === 1 ? 'closeable' : 'default'"
+      (nzOnClose)="onRemoveTag(tag)"
+      [nzColor]="tag.color"
+      style="margin-top: 4px"
+    >
+      {{ sliceTagName(tag) }}
+    </nz-tag>
+    <a (click)="onShowTagsModal()">
+      <nz-tag style="margin-top: 4px">
+        <i nz-icon nzType="plus"></i>
+        {{ 'tag.new' | i18n }}
+      </nz-tag>
+    </a>
+  </nz-form-control>
+</nz-form-item>
+
+<nz-form-item *ngIf="item.type === 'textarea-limit'">
+  <nz-form-label nzSpan="7" [nzRequired]="item.required" [nzFor]="item.field" [nzTooltipTitle]="item.tooltip"
+    >{{ item.name }}
+  </nz-form-label>
+  <nz-form-control [nzSpan]="8">
+    <nz-textarea-count [nzMaxCharacterCount]="extra.textarea_limit || 100">
+      <textarea
+        [(ngModel)]="value"
+        (ngModelChange)="onChange($event)"
+        rows="3"
+        nz-input
+        [name]="item.field"
+        [id]="item.field"
+        [placeholder]="item.placeholder ? item.placeholder : ''"
+      ></textarea>
+    </nz-textarea-count>
+  </nz-form-control>
+</nz-form-item>
+
+<!-- 选择TAG弹出框 -->
+<nz-modal
+  [(nzVisible)]="isManageModalVisible"
```

Comment:
```suggestion
<nz-modal
  *ngIf="item.type === 'tags-selection'"
  [(nzVisible)]="isManageModalVisible"
```
hi, thanks lgtm. Suggest add ngif to avoid rendering unused modal every time.